tracardi/process_engine/action/v1/return_random_element_action.py

```diff
@@ -0,0 +1,88 @@
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.action_runner import ActionRunner
+from pydantic import BaseModel, validator
+from typing import List
+from random import randint
+from tracardi.service.plugin.domain.result import Result
+
+
+class Config(BaseModel):
+    list_of_items: List[str]
+
+    @validator("list_of_items")
+    def validate_list_of_items(cls, value):
+        if not value:
+            raise ValueError("This list have to contain at least one element.")
+        return value
+
+
+def validate(config: dict) -> Config:
+    return Config(**config)
+
+
+class RandomReturner(ActionRunner):
+
+    def __init__(self, **kwargs):
+        self.config = Config(**kwargs)
+
+    async def run(self, payload):
+        dot = self._get_dot_accessor(payload)
+
+        random_value_dot = self.config.list_of_items.pop(randint(0, len(self.config.list_of_items) - 1))
+        random_value = dot[random_value_dot]
+
+        return Result(port="random_element", value={"value": random_value})
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        spec=Spec(
+            module=__name__,
+            className='RandomReturner',
+            inputs=["payload"],
+            outputs=["random_element"],
+            version='0.6.0.1',
+            license="MIT",
+            author="Dawid Kruk",
+            manual="random_element_action",
+            init={
+                "list_of_items": []
+            },
+            form=Form(
+                name="Plugin configuration",
+                groups=[
+                    FormGroup(
+                        fields=[
+                            FormField(
+                                id="list_of_items",
+                                name="List of items",
+                                description="Provide a list of paths to values that plugin will choose randomly from.",
+                                component=FormComponent(type="listOfDotPaths", props={
+                                    "label": "List",
+                                    "defaultMode": 2
+                                })
+                            )
+                        ]
+                    )
+                ]
+            )
+
+        ),
+        metadata=MetaData(
+            name='Return random item',
+            desc='Returns a random value from list given in configuration.',
+            type='flowNode',
```

Comment:
The type has a default value so you do not have to provide it. It is better to not provide it if it is the default.